### ISSUE TITLE
Invalid point encountered!

### ISSUE THREAD

**Issue by theTB** [AtomLinter/linter-flake8#266]
I keep getting the following error pop up. Keeps popping up at incomplete lines of code I am still writing. Any suggestions as to what this is?

```
Error: linter-flake8:: Invalid point encountered! See console for details.
Error: linter-flake8:: Invalid point encountered! See console for details.
    at Error (native)
    at extractRange (/Users/tbansil/.atom/packages/linter-flake8/lib/main.js:73:11)
    at Object.<anonymous> (/Users/tbansil/.atom/packages/linter-flake8/lib/main.js:221:25)
    at undefined.next (native)
    at step (/Users/tbansil/.atom/packages/linter-flake8/lib/main.js:9:273)
```

**Comment by Arcanemagus** [comment]
If you check the console what is the error code the message was attempting to mark? It's likely this is a duplicate of #256.

**Comment by silentsokolov** [comment]
I faced a similar error:

![1477397412157](https://cloud.githubusercontent.com/assets/398939/19685614/2ec1e04a-9ac6-11e6-816d-b9ed5b763af7.jpg)

**Comment by Arcanemagus** [comment]
@silentsokolov As I guessed when you posted over in #265, and can now confirm from that screenshot you are _definitely_ hitting the bug in `flake8` that was initially reported in #256.

**Comment by wonderbeyond** [comment]
Me too.
![image](https://cloud.githubusercontent.com/assets/228361/19756218/9b6300de-9c4d-11e6-9ab1-12515115c5e8.png)

**Comment by BrunoBernardino** [comment]
This is happening very frequently, after the update before the last one, it's tempting me to just disable it :/

**Comment by ashcrow** [comment]
I'm having the same issue. I've disabled it for now but am looking forward to re-enabling this plugin. It's extremely helpful!

**Comment by can-z** [comment]
My PR #271 seems to at least suppress the errors for now although it might not be accepted as the ultimate fix. You should be able to paste my changes into your local file to fix the issue.

**Comment by Arcanemagus** [comment]
v2.1.0 has been released which moves the error reporting to a Linter message.

**Comment by Arcanemagus** [comment]
Also it looks like the fix for the originally reported issue has been released in `flake8==3.2.1` 🎉
